jerry-core/ecma/builtin-objects/ecma-builtin-internal-routines-template.inc.h

```diff
@@ -284,21 +325,21 @@ TRY_TO_INSTANTIATE_PROPERTY_ROUTINE_NAME (BUILTIN_UNDERSCORED_ID) (ecma_object_t
  * @return string values collection
  */
 void
-LIST_LAZY_PROPERTY_NAMES_ROUTINE_NAME (BUILTIN_UNDERSCORED_ID) (ecma_object_t *object_p, /**< a built-in object */
-                                                                /** true -  list enumerable properties
-                                                                 *           into main collection,
-                                                                 *           and non-enumerable to
-                                                                 *           collection of 'skipped
-                                                                 *           non-enumerable'
-                                                                 *           properties,
-                                                                 *   false - list all properties into
-                                                                 *           main collection.
-                                                                 */
-                                                                bool separate_enumerable,
-                                                                /** 'main' collection */
-                                                                ecma_collection_header_t *main_collection_p,
-                                                                /** skipped 'non-enumerable' collection */
-                                                                ecma_collection_header_t *non_enum_collection_p)
+LIST_LAZY_PROPERTY_NAMES_ROUTINE_NAME (ecma_object_t *object_p, /**< a built-in object */
+                                       /** true -  list enumerable properties
+                                        *           into main collection,
+                                        *           and non-enumerable to
+                                        *           collection of 'skipped
+                                        *           non-enumerable'
+                                        *           properties,
+                                        *   false - list all properties into
+                                        *           main collection.
+                                        */
+                                       bool separate_enumerable,
+                                       /** 'main' collection */
+                                       ecma_collection_header_t *main_collection_p,
```

Comment:
Fix argument comment to the correct doxygen style: `ecma_collection_header_t *main_collection_p, /**< 'main' collection */`